.gitignore

```diff
@@ -1,8 +1,11 @@
 node_modules/
 *.log
-yarn.lock
 package-lock.json
 .idea
+config/render/startConfig.json
+client-stats.json
```

Comment:
Should these be added to consumer's gitignores via `configure`?

Reply:
I can remove `client-stats.json`, I used it as a debugging tool but it shouldn't generate any more. And `startConfig` isn't required as it will always live within `node_modules` for consumers.